src/renderer/components/+custom-resources/crd-details.tsx

```diff
@@ -143,11 +146,12 @@ export class CRDDetails extends React.Component<Props> {
         {validation &&
         <>
           <DrawerTitle title="Validation"/>
-          <AceEditor
-            mode="yaml"
-            className="validation"
+          <MonacoEditor
+            options = {{readOnly: true, ...UserStore.getInstance().getEditorOptions()}}
+            className={cssNames( "MonacoEditor", "validation")}
```

Comment:
Please remove the space after `cssNames( ` here and in other MonacoEditor instances.